example/Cpp/src/AlarmClock/AlarmClock.cmake

```diff
@@ -0,0 +1,8 @@
+find_package (Threads)
+
+set(CROW_HEADERS ../../Crow/include/)
```

Comment:
There is no Crow directory checked in. In consequence, this doesn't resolve to anything useful.

Reply:
But I need the headers to be included. (target_include_directory) 

I think it's better to keep it and when somebody want's to try it out they should go to the main repo.

Reply:
In the main repo I have crow as a submodule. 

I am not super familiar  with the in depth details of git submodules. I don't think we can add the submodule hidden inside the example without polluting the root .gitmodule. 

Your last proposal I like personally the most. When I remember correctly I didn't do this because I would have to package that library in nix to make that really work smoothly.